postman.json

```diff
@@ -1452,24 +1612,23 @@
 							}
 						],
 						"url": {
-							"raw": "http://transmorpher.test/images/{{user}}/{{imageIdentifier}}/q-123",
-							"protocol": "http",
+							"raw": "{{domain}}/api/v1/media/{{imageIdentifier}}/versions",
 							"host": [
-								"transmorpher",
-								"test"
+								"{{domain}}"
 							],
 							"path": [
-								"images",
-								"{{user}}",
+								"api",
+								"v1",
+								"media",
 								"{{imageIdentifier}}",
-								"q-123"
+								"versions"
 							]
 						}
 					},
 					"response": []
 				},
 				{
-					"name": "Derivative: invalid format parameter",
+					"name": "List Versions: OK",
```

Comment:
We should probably add this to the video api calls too, else it might seem that this api call is not available for videos.